portfolio/src/main/webapp/script.js

```diff
@@ -32,6 +32,21 @@ document.addEventListener('DOMContentLoaded', async function() {
   else {
     commentSectionContainer.style.display = 'block';
   }
+
+//   Hide comment form if the user is not logged-in
```

Comment:
This listener is getting pretty big, I think it might be time to split it into subfunctions